.github/workflows/main.yml

```diff
@@ -29,26 +29,27 @@ jobs:
         echo "created=$(date --utc +%Y-%m-%dT%H:%M:%SZ)" >> $GITHUB_OUTPUT
 
     - name: Set up QEMU
-      uses: docker/setup-qemu-action@v3
+      uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
```

Comment:
Docker is borderline if we should trust them. We've used the hash just in case since it is not really a burden.

So this is fine.